android/app/src/main/java/org/fossasia/openevent/adapters/SpeakersListAdapter.java

```diff
@@ -103,7 +104,7 @@ public void onBindViewHolder(ViewHolder.Viewholder holder, int position) {
 
         holder.getTxtView2().setText(String.format("%s %s", current.getPosition(), current.getOrganisation()));
         holder.getTxtView1().setText(TextUtils.isEmpty(current.getName()) ? "" : current.getName());
-
+        holder.getTxtView3().setText(String.format("%s %s", current.getPosition(), current.getCountry()));
```

Comment:
@mananwason Actually, the position is not being displayed in textview2. So, there is no need of this line.

    holder.getTxtView2().setText(String.format("%s %s", current.getPosition(), current.getOrganisation()));

Should I change it with:

    holder.getTxtView2().setText(TextUtils.isEmpty(current.getOrganisation()) ? "" : current.getOrganisation());